<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserServiceMatrixWindowUi</class>
 <widget class="QMainWindow" name="UserServiceMatrixWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mCentralWidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mUserFilterEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Service Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mServiceFilterEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
